### PR TITLE
Handled transferFrom and safeTransferFrom

### PR DESCRIPTION
**Pull request by zincoshine**

### CODE REVIEW

File: src/services/processTx.js [hunk 1]
```diff
@@ -108,7 +108,11 @@ async function newPendingTran(tx, protocol) {
             // smart contract call hence the asset must be the token name
             to = data.params[0].value;
             pillarId = await client.getAsync(to);
-            [, { value }] = data.params;
+            value = data.params[1].value;
```
Comment (ricardoul) [inline src/services/processTx.js]: Is necessary this change?

File: src/services/processTx.js [hunk 2]
```diff
@@ -108,7 +108,11 @@ async function newPendingTran(tx, protocol) {
...
+          } else if (data.name === 'transferFrom' || data.name === 'safeTransferFrom') {
+            to = data.params[1].value;
+            pillarId = await client.getAsync(to);
+            value = data.params[2].value;
```
Comment (ricardoul) [inline src/services/processTx.js]: this one could be  [, ,  { value }] = data.params;

File: src/services/processTx.js (hunk 1 above)
Comment (zincoshine) [reply src/services/processTx.js]: I didnt realize that this change was related to linter, will amend it.

File: src/services/processTx.js (hunk 2 above)
Comment (zincoshine) [reply src/services/processTx.js]: Pls see earlier, will amend.

**Pull request merged**